macros/parsers/parserCheckboxList.pl

```diff
@@ -409,7 +441,8 @@ sub labelFormat {
 sub getIndexByValue {
 	my ($self, $value) = @_;
 	return -1 unless defined $value;
-	my ($index) = grep { $self->{values}[$_] eq $value } 0 .. $#{ $self->{values} };
+	my ($index) =
+		grep { $self->{values}[$_] eq $value } 0 .. $#{ $self->{values} };
```

Comment:
I definitely do not like this change.  This without a doubt is more readable on one line.